pkg/kong/consumer_test.go

```diff
@@ -0,0 +1,36 @@
+package kong
+
+import (
+	"github.com/api7/kong-to-apisix/pkg/utils"
+	"github.com/stretchr/testify/assert"
+	"testing"
```

Comment:
Please use goimport to sort it like other files, see https://github.com/api7/kong-to-apisix/blob/2c941302103026edb508cedea10c7985c54849cd/pkg/utils/utils.go#L4